pkg/types/aws/defaults/platform.go

```diff
@@ -6,6 +6,7 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/aws"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
This should be above this block to respect the import order in the installer project. See https://github.com/openshift/installer/blob/main/.golangci.yaml#L103-L107